backends/tc/ebpfCodeGen.cpp

```diff
@@ -124,12 +124,41 @@ void PNAEbpfGenerator::emitP4TCFilterFields(EBPF::CodeBuilder *builder) const {
     builder->emitIndent();
     builder->appendFormat("__u16 prio");
     builder->endOfStatement(true);
-
+    emitP4TCActionParam(builder);
     builder->blockEnd(false);
     builder->endOfStatement(true);
     builder->newline();
 }
 
+void PNAEbpfGenerator::emitP4TCActionParam(EBPF::CodeBuilder *builder) const {
+    std::vector<cstring> actionParamList;
+    for (auto table : tcIR->tcPipeline->tableDefs) {
+        if (table->isTcMayOverride) {
+            cstring tblName = table->getTableName();
+            cstring defaultActionName = table->defaultMissAction->getActionName();
+            auto actionNameStr = defaultActionName.c_str();
+            for (long unsigned int i = 0; i < defaultActionName.size(); i++) {
```

Comment:
Why use long unsigned int? long and int are both 4 bytes

Reply:
Compiler issues warning because of the return type of size() function